src/Components/NavBar/Menus/NewMobileNavMenu.tsx

```diff
@@ -33,13 +45,63 @@ export const NewMobileNavMenu: React.FC<Props> = props => {
             <MobileSubmenuLink menu={(artists as MenuLinkData).menu}>
               {(artists as MenuLinkData).menu.title}
             </MobileSubmenuLink>
-            <MobileLink href="/auctions">Auctions</MobileLink>
-            <MobileLink href="/articles">Editorial</MobileLink>
-            <MobileLink href="/galleries">Galleries</MobileLink>
-            <MobileLink href="/fairs">Fairs</MobileLink>
-            <MobileLink href="/shows">Shows</MobileLink>
-            <MobileLink href="/institutions">Museums</MobileLink>
-            <MobileLink href="/gallery-partnerships">
+            <MobileLink
+              href="/auctions"
+              onClick={() => {
+                handleClickTracking("/auctions", "Auctions")
+              }}
+            >
+              Auctions
+            </MobileLink>
+            <MobileLink
+              href="/articles"
+              onClick={() => {
+                handleClickTracking("/articles", "Editorial")
+              }}
+            >
+              Editorial
+            </MobileLink>
+            <MobileLink
+              href="/galleries"
+              onClick={() => {
+                handleClickTracking("/galleries", "Galleries")
+              }}
+            >
+              Galleries
+            </MobileLink>
+            <MobileLink
+              href="/fairs"
+              onClick={() => {
+                handleClickTracking("/fairs", "Fairs")
+              }}
+            >
+              Fairs
+            </MobileLink>
+            <MobileLink
+              href="/shows"
+              onClick={() => {
+                handleClickTracking("/shows", "Shows")
+              }}
+            >
+              Shows
+            </MobileLink>
+            <MobileLink
+              href="/institutions"
+              onClick={() => {
+                handleClickTracking("/institutions", "Museums")
+              }}
+            >
+              Museums
+            </MobileLink>
+            <MobileLink
```

Comment:
Instead of having an explicit `onClick` call on each of these instances of `<MobileLink>`, could we try putting the click/tracking handler on the `<MobileLink>` component directly?

It looks like most of the custom properties (`destination_path`, `subject`) should be present there anyways, and where not perhaps we can pass that through from a parent component?

If possible to do so, it could make this code a bit more concise (and also make it less likely for someone adding a new link to forget to add tracking in the future!).